src/tls13.c

```diff
@@ -14365,6 +14365,7 @@ int wolfSSL_read_early_data(WOLFSSL* ssl, void* data, int sz, int* outSz)
         if (ret > 0)
             *outSz = ret;
         if (ssl->error == ZERO_RETURN) {
+            *outSz = 0;
```

Comment:
Maybe it would be better to do this at the very beginning of the function in all cases?

Reply:
My rationale was that if the function errors out then it doesn't change the parameter. 
But I'm ok moving that at the beginning as well.